tensorflow_datasets/core/utils/gcs_utils.py

```diff
@@ -37,8 +37,10 @@
 GCS_DATASETS_DIR = 'datasets'
 
 
-_is_gcs_disabled = False
```

Comment:
I think you could replace this by `_is_gcs_disabled = os.name == 'nt'`

Reply:
That Global variable seemed unnecessary